templates/phoenix_elixir/files/config/config.exs

```diff
@@ -0,0 +1,30 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Mix.Config module.
+#
+# This configuration file is loaded before any dependency and
+# is restricted to this project.
+
+# General application configuration
+use Mix.Config
+
+config :template_elixir,
+  ecto_repos: [TemplateElixir.Repo]
+
+# Configures the endpoint
+config :template_elixir, TemplateElixirWeb.Endpoint,
+  url: [host: "localhost"],
+  secret_key_base:   System.get_env("PLATFORM_PROJECT_ENTROPY"),
```

Comment:
Huh.  Also redundant?  Why does this get set 3 times?

Reply:
I think actually that in the end
- set in app.yaml
- read in config.exs

is sufficient